src/dtclient/client.go

```diff
@@ -107,6 +107,9 @@ const (
 	TokenScopeDataExport        = "DataExport"
 	TokenScopeReadConfig        = "ReadConfig"
 	TokenScopeWriteConfig       = "WriteConfig"
+	TokenScopeMetricsIngest     = "metrics.ingest"
```

Comment:
why are these tokens in a different format than the older ones?

Reply:
its not up to me :)